Comment-Code-Poltergeists/nutshell
# Login test branch

# Description

We now have login/register functionality!!!

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

git fetch
gc login-test-branch
open a NEW tab
check that the login modal automatically shows
click register and check that it changes to a register form
leave some inputs empty and check that it gives you a window alert
try registering with an email thats already been used and check that it gives you a window alert
login with a user that you already have and check that modal hides and session storage id is 
    set(you will probably have to reload page to see all content)
open another tab and register a new account(check that it gives you a window alert if passwords do not match)



# Checklist:
